### DISCUSSION TITLE
tldraw can be used Vue app?

### DISCUSSION THREAD

**Discussion by yulafezmesi** [tldraw/tldraw-v1#555] [Q&A] [answered]
Hello, 

Can I use tldraw separately existing javascript project?

**Comment by steveruizok** [comment, marked as answer]
No, tldraw only works in React. If you're not able to run both, then your best bet is to have a separate React app for the tldraw editor and use an iframe to get it into your app.

**Reply by elrumo** [reply]
Have you considered creating a web component? That way it would be able to run pretty much anywhere, even in vanilla JS, and it wouldn't require any extra development and porting efforts from you.
